src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/decommission.go

```diff
@@ -38,31 +45,41 @@ leader handles the request.
 			cl, err := admin.NewClient(fs, cfg)
 			out.MaybeDie(err, "unable to initialize admin client: %v", err)
 
-			brokers, err := cl.Brokers(cmd.Context())
-			out.MaybeDie(err, "unable to get broker list: %v", err)
+			if !force {
+				brokers, err := cl.Brokers(cmd.Context())
+				out.MaybeDie(err, "unable to get broker list: %v; to bypass the node version check re-run this with --force", err)
 
-			var b admin.Broker
-			var found bool
-			for _, br := range brokers {
-				if br.NodeID == broker {
-					b, found = br, true
-					break
-				}
-			}
-			if !found {
-				out.Die("unable to find broker %v in the cluster", broker)
-			}
+				var (
+					b             admin.Broker
+					found, anyOld bool
+				)
+				for _, br := range brokers {
+					if br.NodeID == broker {
+						version, err := redpanda.VersionFromString(br.Version)
+						out.MaybeDie(err, "unable to get broker %q version: %v; to bypass the node version check re-run this with --force", br.NodeID, err)
```

Comment:
I think we should reference our long help text actually as well, which explains why we have --force. Wdyt about 

```
; read this command's help text for how to bypass the node version check
```

or

Similar thought to below
```
to bypass the node version check re-run this with --force; see this commands help text for for more details
```